docs/guides/adding-component-types.md

```diff
@@ -140,7 +140,7 @@ the repository. There is an issue tracking curring status of this effort,
 
 - If it's not on [the storybook](https://react.carbondesignsystem.com), it's
   probably not public.
-- The entire Public API is snapshotted, you can
+- The entire Public API is snapshot-ed, you can
```

Comment:
I believe this is the correct past tense of "snapshot"

<img width="608" alt="Screenshot 2023-02-23 at 2 58 51 PM" src="https://user-images.githubusercontent.com/11928039/221016612-fc730b28-134b-45a4-b0d5-985e281f0879.png">

Reply:
My bad; the spell-checking plugin I was using didn't seem to like this. Fixed.